config/params.php

```diff
@@ -72,10 +73,13 @@
         'schema-providers' => [],
 
         /**
-         * Config for {@see \Yiisoft\Yii\Cycle\Schema\Conveyor\AnnotatedSchemaConveyor}
-         * Annotated entity directories list.
+         * Annotated/attributed entity directories list.
          * {@see \Yiisoft\Aliases\Aliases} are also supported.
          */
+        'entity-paths' => [],
+        'conveyor-class' => CompositedSchemaConveyor::class,
```

Comment:
What about `conveyor`?

Reply:
i think  it also can be replaced to aliases like this:

`'conveyor-schema-source' => 'attributes', // 'annotations' 'all' ('both')`

Reply:
Why? If add string value, then need class with constants. Class name is fine for me.

Reply:
Why? If use string values, then need create class with constants. Class names is OK.

Reply:
@vjik what about parameter name? `conveyor-class` or `conveyor`?

Reply:
> @vjik what about parameter name? `conveyor-class` or `conveyor`?

I think `conveyor` good.

Reply:
renamed